examples/src/main/scala/za/co/absa/atum/utils/SparkJobRunnerMethods.scala

```diff
@@ -30,7 +30,7 @@ trait SparkJobRunnerMethods {
     val jobClassSymbol = universe runtimeMirror jobClass.getClassLoader classSymbol jobClass
     val jobInstance =
       if (jobClassSymbol.isModuleClass) jobClass getField "MODULE$" get jobClass
```

Comment:
I know not your code, but this form is hard to read and not bringing other value, IMHO. Would change it to more standard format.
```suggestion
   if (jobClassSymbol.isModuleClass) jobClass.getField("MODULE$").get(jobClass)      
```

Optional bonus put the `if` branches into curly braces. Like:
```
if (...) {
  ...
} else {
  ...
}
```